analyze.go

```diff
@@ -1836,6 +1868,9 @@ var directives = map[string][]uint{
 	"upstream_conf": {
 		ngxHTTPLocConf | ngxConfNoArgs,
 	},
+	"usage_report": {
+		ngxMgmtMainConf | ngxConfAny,
```

Comment:
NGINX does accept `usage_report;`

Reply:
Then should it be `ngxConfNoArgs | ngxConfTake12`?